crates/cac_toml/Cargo.toml

```diff
@@ -17,4 +17,4 @@ path = "src/bin.rs"
 clap = { version = "4.3.0", features = ["derive"] }
 pest = "2.6"
 pest_derive = "2.6"
-toml = { version = "0.8.8", features = ["preserve_order"] }
+toml = { workspace = true }
```

Comment:
Will this pick-up the `preserve_order` if someone just includes `cac_toml` outside of this workspace?  Could we validate that?

Reply:
yes this will set the features for the crate, in the same way it is specified in the workspace `Cargo.toml`
have verified this